armi/cases/case.py

```diff
@@ -855,7 +855,7 @@ def _copyInputsHelper(
     -------
     destFilePath (or origFile) : str
     """
-    sourceName = os.path.basename(sourcePath)
+    sourceName = pathlib.Path(sourcePath).name
```

Comment:
this does the same thing, but `pathlib` is robust to a trailing `/` whereas `os.path` is not.